Sources/MapboxMaps/Annotations/Generated/CircleAnnotation.swift

```diff
@@ -129,6 +135,33 @@ public struct CircleAnnotation: Annotation {
         }
     }
 
+
+    /// Get the offset geometry for the touch point
+    func getOffsetGeometry(_ mapboxMap: MapboxMap, moveDistancesObject: MoveDistancesObject?) -> Geometry? {
+        let maxMercatorLatitude = 85.05112877980659
+        let minMercatorLatitude = -85.05112877980659
+
+        guard let moveDistancesObject = moveDistancesObject else {
+          return nil
+        }
+
+           let point = point.coordinates
```

Comment:
Nit: Rename is needed as there is more than one point in the context. Consider `startPoint` name.